src/style.css

```diff
@@ -0,0 +1,36 @@
+body,
+h1,
+p,
+ul,
+li,
+main {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  position: absolute;
+  display: flex;
+  /* border: thick double #32a1ce; */
+  width: 140px;
+  height: 160px;
```

Comment:
- Don't fix container size (if there is no such a requirement)
![image](https://github.com/mate-academy/layout_stars/assets/63243136/f5e9bf29-ae31-434c-af34-ed6fd807e6e7)